content/docs/addons-test-utils.md

```diff
@@ -40,17 +40,17 @@ var ReactTestUtils = require('react-dom/test-utils'); // ES5 with npm
  - [`renderIntoDocument()`](#renderintodocument)
  - [`Simulate`](#simulate)
 
-## Reference {#reference}
+## Referanslar {#reference}
```

Comment:
I'm not 100% sure but it feels like we can use `Kaynakça` here. Otherwise let's keep this single as `Referans`

Reply:
Changed it as `Referans`.